### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by sdaveas**

### CODE REVIEW

**Review by milapsheth — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -86,24 +93,56 @@ OPTIONS:
...
+
+In containerized environments, the `auto` mnemonic command can be used. `auto` works as follows:
+1. If no `import` file exists in `TOFND_HOME`, then `tofnd` attempts to create a new mnemonic. 
+    1. If an mnemonic is already stored in the kv store, then the creation is aborted, and the existing mnemonic is used.
+    2. If no mnemonic is stored in the kv store, then a fresh mnemonic is created, and the mnemonic is exported. The exported file is then renamed to `TOFND_HOME/import`.
+2. If an `import` file exists in `TOFND_HOME`, then `tofnd` attempts to use it. 
+    1. If a mnemonic is already stored in the kv store, then the import is aborted, and the existing mnemonic is used.
+    2. If no mnemonic is stored in the kv store, then the `import` file is used to store the mnemonic in the kv store.
+
+In short, `auto` implies the following order: 1) try to use existing mnemonic, 2) try to import a mnemonic from file, 3) try create a new mnemonic.
```
Comment (ggutoski) [inline README.md:139]:
```suggestion
In containerized environments the `auto` mnemonic command can be used.  This command is implemented in `entrypoint.sh` and does the following:
1. Try to use existing mnemonic.  If successful then launch `tofnd` server.
2. Try to import a mnemonic from file.  If successful then launch `tofnd` server.
3. Create a new mnemonic.  The newly created mnemonic is automatically written to the file `TOFND_HOME/export`---rename this file to `TOFND_HOME/import` so as to unblock future executions of tofnd.  Then launch `tofnd` server.
```

These instructions are pretty complicated.  How about this instead?  It's unfortunate that `entrypoint.sh` doesn't actually act exactly as described above but the effect is the same.  I don't think it's worth describing what `entrypoint.sh` actually does.  But I do wonder whether it's worth changing `entrypoint.sh` to somehow behave exactly as this suggestion describes.

File: README.md [hunk 2]
```diff
@@ -86,24 +93,56 @@ OPTIONS:
...
+
+The rationale behind `auto` is that users can fritionalessly run and restart their tofnd nodes without having to run multiple commands.
```
Comment (ggutoski) [inline README.md:141]:
```suggestion
The rationale behind `auto` is that users can frictionlessly launch and restart their tofnd nodes without the need to execute multiple commands.
```

**Review by ggutoski — COMMENTED** [review]
Your recent changes to `entrypoint.sh` seem to be missing.  Need to restore those changes before merging.

File: README.md (hunk 1 above)
Comment (sdaveas) [reply README.md:139]: Yeah I guess I got carried away. I like narrowing down to simpler steps. BTW, by changing the order to 2 -> 3 -> 1 you describe exactly how `entrypoint.sh` uses `auto`.

**Comment by sdaveas** [comment]
> Your recent changes to `entrypoint.sh` seem to be missing. Need to restore those changes before merging.

`entrypoint.sh` is recovered in `new-mnemonic-spec` branch

**Pull request merged**